webrtc.html

```diff
@@ -524,26 +528,31 @@
 
           <li>
             <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            execute the function from the front of the queue.</p>
+          </li>
+
+          <li>
+            <p>When the promise returned by the function is settled,
```

Comment:
"Upon fulfillment or rejection of the promise returned by the function," - [§ 5.3. Reacting to Promises](http://www.w3.org/2001/tag/doc/promises-guide#shorthand-reacting).